src/main/java/internet/shop/exception/NoConnectException.java

```diff
@@ -0,0 +1,9 @@
+package internet.shop.exception;
+
+import java.sql.SQLException;
+
+public class NoConnectException extends RuntimeException {
```

Comment:
you've put a checkmark that you reviewed your hw according to checklist, but here is still a mistake about exception from it, be more attentive next time 
![image](https://user-images.githubusercontent.com/58385506/104699236-18fe8900-571b-11eb-8693-6ca06bf8d80e.png)

Reply:
okey, done, i will more attentive